### ISSUE TITLE
Unable to install optic cli on macos with brew

### ISSUE THREAD

**Issue by paskos** [opticdev/optic#398]
I tried to follow the brew instructions
I got the following error:

brew install opticdev/optic/api
==> Tapping opticdev/optic
Cloning into '/usr/local/Homebrew/Library/Taps/opticdev/homebrew-optic'...
remote: Enumerating objects: 757484, done.
remote: Total 757484 (delta 0), reused 0 (delta 0), pack-reused 757484
Receiving objects: 100% (757484/757484), 303.19 MiB | 11.07 MiB/s, done.
Resolving deltas: 100% (502921/502921), done.
Tapped 1 formula (28 files, 323.4MB).
==> Installing api from opticdev/optic
==> Downloading https://homebrew.bintray.com/bottles/node-15.0.1.catalina.bottle.tar.gz
==> Downloading from https://d29vzk4ow07wi7.cloudfront.net/9cdf039763f006eef0cc0aaebb13bc8173f017d961ac41c30f26d45a23ecffe0?response-content-dispos
######################################################################## 100.0%
==> Downloading https://registry.npmjs.org/@useoptic/cli/-/cli-8.4.2.tgz

curl: (22) The requested URL returned error: 404
Error: Failed to download resource "api"
Download failed: https://registry.npmjs.org/@useoptic/cli/-/cli-8.4.2.tgz

**Comment by acunniffe** [comment]
Hey @paskos -- I was able to reproduce this today. We pulled 8.4.2 from NPM earlier this week shortly after publishing. We're working to integrate the new Rust diff engine and it's been changing a lot about how the app builds. 

When it was unpublished it looks like the brew reference to the unpublished version wasn't removed. This is good for us to know going forward in case we ever have to pull a build again. 

For now, try using yarn or npm 

I'll get a new brew release out this weekend to fix the issue and document this known issue. We probably shouldn't pull stuff in the future from NPM, we only did it because it was out for less than an hour at the time, but can definitely see how that choice can ripple out to other system. 

thanks! Let us know if there's anything else we can help with.
